### ISSUE TITLE
Can't build 3.4.18

### ISSUE THREAD

**Issue by John-Colvin** [libmir/mir-algorithm#198]
```
C:\Users\jcolvin_sym\AppData\Local\dub\packages\mir-algorithm-3.4.18\mir-algorithm\source\mir\series.d(2514,13): Error: expression expected, not ref
C:\Users\jcolvin_sym\AppData\Local\dub\packages\mir-algorithm-3.4.18\mir-algorithm\source\mir\series.d(2514,18): Error: expression expected, not scope
C:\Users\jcolvin_sym\AppData\Local\dub\packages\mir-algorithm-3.4.18\mir-algorithm\source\mir\series.d(2514,24): Error: found ref when expecting ,
C:\Users\jcolvin_sym\AppData\Local\dub\packages\mir-algorithm-3.4.18\mir-algorithm\source\mir\series.d(2514,33): Error: expression expected, not scope
C:\Users\jcolvin_sym\AppData\Local\dub\packages\mir-algorithm-3.4.18\mir-algorithm\source\mir\series.d(2514,39): Error: found return when expecting ,
C:\Users\jcolvin_sym\AppData\Local\dub\packages\mir-algorithm-3.4.18\mir-algorithm\source\mir\series.d(2514,46): Error: expression expected, not ref
C:\Users\jcolvin_sym\AppData\Local\dub\packages\mir-algorithm-3.4.18\mir-algorithm\source\mir\series.d(2514,50): Error: found left when expecting ,
C:\Users\jcolvin_sym\AppData\Local\dub\packages\mir-algorithm-3.4.18\mir-algorithm\source\mir\series.d(2514,56): Error: found => when expecting ) following template argument list
C:\Users\jcolvin_sym\AppData\Local\dub\packages\mir-algorithm-3.4.18\mir-algorithm\source\mir\series.d(2514,59): Error: unexpected identifier left in declarator
C:\Users\jcolvin_sym\AppData\Local\dub\packages\mir-algorithm-3.4.18\mir-algorithm\source\mir\series.d(2515,13): Error: identifier expected following comma, not ref
C:\Users\jcolvin_sym\AppData\Local\dub\packages\mir-algorithm-3.4.18\mir-algorithm\source\mir\series.d(2515,17): Error: basic type expected, not (
C:\Users\jcolvin_sym\AppData\Local\dub\packages\mir-algorithm-3.4.18\mir-algorithm\source\mir\series.d(2515,17): Error: function declaration without return type. (Note that constructors are always named this)
C:\Users\jcolvin_sym\AppData\Local\dub\packages\mir-algorithm-3.4.18\mir-algorithm\source\mir\series.d(2515,80): Error: no identifier for declarator ref _error_(ref scope key, return ref scope left, return ref scope right)
C:\Users\jcolvin_sym\AppData\Local\dub\packages\mir-algorithm-3.4.18\mir-algorithm\source\mir\series.d(2515,80): Error: semicolon expected following function declaration
C:\Users\jcolvin_sym\AppData\Local\dub\packages\mir-algorithm-3.4.18\mir-algorithm\source\mir\series.d(2515,80): Error: found => instead of statement
```

**Comment by John-Colvin** [comment]
Sorry, I see it's a compiler version thing.

It would be great if a patch release didn't break compiler compatibility.

**Comment by 9il** [comment]
Sorry about that. Will add a fix tomorrow

**Comment by 9il** [comment]
fixed in v3.4.19
